android/app/src/main/java/org/electroncash/electroncash3/Util.kt

```diff
@@ -94,11 +94,14 @@ fun copyToClipboard(text: CharSequence, what: Int? = null) {
     @Suppress("DEPRECATION")
     (getSystemService(ClipboardManager::class)).text = text
     val message = if (what == null) app.getString(R.string.text_copied)
-                  else app.getString(R.string._s_copied, app.getString(what))
+    else app.getString(R.string._s_copied, app.getString(what))
     toast(message, Toast.LENGTH_SHORT)
```

Comment:
Both changes to this file are just making the whitespace worse. it should be restored to its original content.

Reply:
reverted

Reply:
The change above hasn't been reverted, and now the most recent commit to this file has introduced several other unnecessary whitespace changes.

It would be easier if you let me resolve the conversations, that way I can keep track of which things I've checked.

Reply:
Also, I made a comment above about whitespace changes in the .gitignore and Project_Default.xml. If those are reverted as well, then all 3 files should disappear from the "Files changed" tab above.

Reply:
Sorry TIL copying and pasting into Android Studio is bad because it automatically reformats things, will do it from the command line instead.

Reply:
You can alter the indentation rules to match the existing code at File > Settings > Editor > Code Style > Kotlin. In particular, I think you might need to change Tabs and Indents > Continuation indent to 4.